docs/reference/esql/esql-syntax.asciidoc

```diff
@@ -160,14 +160,15 @@ Datetime intervals and timespans can be expressed using timespan literals.
 Timespan literals are a combination of a number and a qualifier. These
 qualifiers are supported:
 
-* `millisecond`/`milliseconds`
-* `second`/`seconds`
-* `minute`/`minutes`
-* `hour`/`hours`
-* `day`/`days`
-* `week`/`weeks`
-* `month`/`months`
-* `year`/`years`
+* `millisecond`/`milliseconds`/`ms`
+* `second`/`seconds`/`s`
+* `minute`/`minutes`/`min`
+* `hour`/`hours`/`h`
+* `day`/`days`/`d`
+* `week`/`weeks`/`w`
+* `month`/`months`/`mo`
+* `quarter`/`quarters`/`q`
+* `year`/`years`/`yr`/`y`
```

Comment:
- Are we sure we want to deviate from the [standard abbreviations](https://www.elastic.co/guide/en/elasticsearch/reference/8.13/common-options.html#date-math) used in ES? Applies to `mo` and `min`. In SQL, apparently there is yet another [set of abbreviations](https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-functions-datetime.html#sql-functions-datetime-add); not sure if we should follow suit, but having 3 different sets of abbreviations will likely be confusing.
- Are we sure that the time period `min` and the aggregation function `min` will never be ambiguous when parsing?

Reply:
> In SQL, apparently there is yet another [set of abbreviations](https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-functions-datetime.html#sql-functions-datetime-add); not sure if we should follow suit, but having 3 different sets of abbreviations will likely be confusing.

Interestingly that you've made this comparison too. I've recently added [some clarifications](https://github.com/elastic/elasticsearch/pull/108301) about the non-overlapping of this set with function parameters.

> Are we sure that the time period min and the aggregation function min will never be ambiguous when parsing?

We have the tests to keep us in check. But I also hope we won't need to change the grammar in a way that there might be conflicts.